pinot-core/src/main/java/org/apache/pinot/core/common/ObjectSerDeUtils.java

```diff
@@ -515,23 +518,23 @@ public VarianceTuple deserialize(ByteBuffer byteBuffer) {
     }
   };
 
-  public static final ObjectSerDe<PinotFourthMoment> PINOT_FOURTH_MOMENT_OBJECT_SER_DE
-      = new ObjectSerDe<PinotFourthMoment>() {
-    @Override
-    public byte[] serialize(PinotFourthMoment value) {
-      return value.serialize();
-    }
+  public static final ObjectSerDe<PinotFourthMoment> PINOT_FOURTH_MOMENT_OBJECT_SER_DE =
+      new ObjectSerDe<PinotFourthMoment>() {
+        @Override
+        public byte[] serialize(PinotFourthMoment value) {
+          return value.serialize();
+        }
 
-    @Override
-    public PinotFourthMoment deserialize(byte[] bytes) {
-      return PinotFourthMoment.fromBytes(bytes);
-    }
+        @Override
+        public PinotFourthMoment deserialize(byte[] bytes) {
+          return PinotFourthMoment.fromBytes(bytes);
+        }
 
-    @Override
-    public PinotFourthMoment deserialize(ByteBuffer byteBuffer) {
-      return PinotFourthMoment.fromBytes(byteBuffer);
-    }
-  };
+        @Override
+        public PinotFourthMoment deserialize(ByteBuffer byteBuffer) {
+          return PinotFourthMoment.fromBytes(byteBuffer);
+        }
+      };
```

Comment:
I am happy to remove the additional formatting - I used the [recommended settings](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#intellij) for IntelliJ  and reformatted every file I touched accordingly.  This does not seem to be enforced on CI so perhaps it is unnecessary.  

What do you think?

Reply:
Not sure if there is an explicit guideline for this repo, but I think it would be better to separate functional and formatting changes into different PRs.

Reply:
I've reverted these additional formatting changes in https://github.com/apache/pinot/pull/11774/commits/0d125dc5682ad0281f1ab1365ec1da731855dcdc